1907-06-17.xml

```diff
@@ -0,0 +1,2605 @@
+                        Steamers "VICTORIA," "PURITAN," and "GERMANIA."</p>
+                    <p>Bi-weekly Express Service between LUXOR and ASSUAN by S.S. "MAYFLOWER."</p>
+                    <p><lb/>Regular weekly Departures to the SECOND CATARACT by the S.S.
+                        "NUBIA."</p>
+                    <p><lb/>THROUGH BOOKINGS TO KHARTOUM, GONDOKORO AND THE WHITE NILE.</p>
+                    <p><lb/>Steamers and Dahabeahs for private charter. Steam Tugs and Steam
+                        Launches for hire.</p>
+                    <p><lb/>FREIGHT SERVICE BY STEAM BARGES BETWEEN CAIRO AND ALEXANDRIA.</p>
+                    <p><lb/>Working in conjunction and under special arrangement with the “Upper
+                        Egypt Hotels Company."</p>
+                    <p>For details and illustrated programmes apply to HEAD OFFICE: Sharia Boulac,
+                        Continental Hotel Buildings," CAIRO</p>
+                    <p>Hamburg-Amerika Linie.</p>
+                    <p>FARES: to NAPLES from £10, to HAMBURG from £17.10.</p>
+                    <table cols="4">
+                        <row>
+                            <cell/>
+                            <cell>Leaves Port Said:</cell>
+                            <cell>Arr. Naples:</cell>
+                            <cell>Arr. Hamburg:</cell>
+                        </row>
+                        <row>
+                            <cell>S.S. Habsburg</cell>
+                            <cell>May 4th</cell>
+                            <cell>May 8th</cell>
+                            <cell>May 20th</cell>
+                        </row>
+                        <row>
+                            <cell>S.S. Rhemania</cell>
+                            <cell>June 15th</cell>
+                            <cell>June 19th</cell>
+                            <cell> July 1st</cell>
+                        </row>
+                        <row>
+                            <cell>S.S. Hohenstanfen</cell>
+                            <cell>July 13th</cell>
+                            <cell>July 17th</cell>
+                            <cell> July 29th</cell>
+                        </row>
+                        <row>
+                            <cell>S.S. Silesia</cell>
+                            <cell>Aug. 10th</cell>
+                            <cell>Aug. 14th</cell>
+                            <cell>Aug. 26th</cell>
+                        </row>
+                    </table>
+                    <p>Monthly connections from Port Said to the ports of the Red Sea and the
+                        Persian Gulf. </p>
+                    <p>Port Said to Colombo, China, and Japan; ay 4th, S.S. Silesia; June 3rd; S.S.
+                        Scandia; July 4th; S.S. Hamburg; August 3rd, S.S. Rhenania. Also occasional
+                        sailing from Port Said to Marseilles, Oran and London. Roomy cabins
```

Comment:
You need two `</div>` tags before starting page 3, in order to close page 2. Then move the `</div>` tag on this line, which closes page 3, to the end of page 3!